setup.py

```diff
@@ -335,6 +335,7 @@ class CheckSDist(sdist_class):
                  'pandas/_libs/index.pyx',
                  'pandas/_libs/algos.pyx',
                  'pandas/_libs/join.pyx',
+                 'pandas/_libs/khash.pyx',
                  'pandas/_libs/indexing.pyx',
                  'pandas/_libs/interval.pyx',
```

Comment:
hashtable is still coupled to khash_python.h

Reply:
I guess I'm still unclear on the transitive-dependencies policy.  A bunch of other things are coupled to hashtable, so are they also coupled to khash_python.h?

Reply:
well this is exactly the issue
you now have an actual module rather than header only so you shouldn’t import from the c files and instead put the defa inside the .pxd

you may want to find an example how to do this
i don’t like the idea of a dummy module

Reply:
Would getting rid of the .pyx file solve this?